### DISCUSSION TITLE
Cannot generate the QR Code in the web page's security settings with the Authenticator app provider

### DISCUSSION THREAD

**Discussion by syneart** [dani-garcia/vaultwarden#4788] [Q&A] [answered]
## Subject of the issue ##

In the web page's security settings, the Provider settings for the Authenticator app in the Two-step login cannot generate a QR Code.

## Deployment environment ##

vaultwarden version: v1.31.0

Install method: Docker - vaultwarden/server:latest

MySQL/MariaDB or PostgreSQL version: MariaDB

## Other relevant details: ##
There is an error when generating the QR code.
<img width="1469" alt="Screenshot 2024-07-27 at 22 35 36" src="https://github.com/user-attachments/assets/67d732d3-49ec-4369-91be-5b519f755d7d">

## Expected behaviour ##
A QR code needs to be generated.

**Comment by BlackDex** [comment, marked as answer]
Re-open that panel, it is a known issue which has been solved  with a newer web-vault already if I'm correct. Only we need some extra checks to be done for the newer web-vault to work correctly.

It's not a issue of Vaultwarden it self.
